docs/src/components/banner/TableOfContentsBanner.tsx

```diff
@@ -1,57 +1,68 @@
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import Link from 'docs/src/modules/components/Link';
-import ROUTES from 'docs/src/route';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
+import { alpha } from '@mui/material/styles';
 
 export default function TableOfContentsBanner() {
   return FEATURE_TOGGLE.enable_toc_banner ? (
     <Link
-      href={ROUTES.home} // Fix me!
+      href="https://ukraine.ua/news/stand-with-ukraine/" // Fix me!
       target="_blank"
       sx={(theme) => ({
         mb: 2,
         p: 1,
         display: 'flex',
         flexDirection: 'column',
         alignItems: 'start',
-        background:
+        backgroundColor:
           theme.palette.mode === 'dark'
-            ? `linear-gradient(90deg, ${theme.palette.primary[900]}, ${theme.palette.primary[600]} 120%)`
-            : `linear-gradient(-90deg, ${theme.palette.primary[700]}, ${theme.palette.primary[500]} 120%)`,
+            ? alpha(theme.palette.primary[900], 0.3)
+            : theme.palette.primary[50],
+        border: '1px solid',
+        borderColor:
+          theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[200],
         borderRadius: 1,
         transitionProperty: 'all',
         transitionTiming: 'cubic-bezier(0.4, 0, 0.2, 1)',
-        transitionDuration: '200ms',
+        transitionDuration: '150ms',
         '&:hover, &:focus-visible': {
-          boxShadow:
+          backgroundColor:
             theme.palette.mode === 'dark'
-              ? '1px 1px 20px 0 rgb(2 2 2 / 50%)'
-              : '1px 1px 20px 0 rgb(90 105 120 / 30%)',
+              ? alpha(theme.palette.primary[900], 0.6)
+              : alpha(theme.palette.primary[100], 0.4),
+          borderColor:
+            theme.palette.mode === 'dark'
+              ? theme.palette.primaryDark[500]
+              : theme.palette.primary[200],
         },
       })}
     >
-      <Typography component="span" variant="body1" fontWeight="bold" sx={{ color: '#fff' }}>
-        Fix me!
+      <Typography component="span" variant="caption" fontWeight="500" color="text.secondary">
+        🇺🇦&#160;&#160;MUI stands in solidarity with the Ukrainian people against the Russian
```

Comment:
This only works on macOS, so only for ~30% of our visitors

<img width="419" alt="Screenshot 2022-03-02 at 15 12 54" src="https://user-images.githubusercontent.com/3165635/156378190-ee9e0d3c-c29a-4fd4-a1e2-200bd3a0301f.png">

Reply:
Oh, thanks for pointing it out. Pushed a little creative iteration to still have the Ukraine flag somehow.